<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1365</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1365</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1365</width>
    <height>768</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sofia Pro</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(24, 24, 24);
color: white;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="detect_btn">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>500</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:gray;
background: #3B3B3B;
height: 50px;
</string>
    </property>
    <property name="text">
     <string>Detect from file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename_text">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>500</y>
      <width>450</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>FileName.mp4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>250</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>80</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>hmi-images/white_corematic_logo_inline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="menu_dashboard">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>250</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border: 2px solid transparent;
border-bottom-color: #ffbd00;


</string>
    </property>
    <property name="text">
     <string>DASHBOARD</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip1_text">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>450</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>192.168.0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>FileName.mp4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cam1_btn">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>300</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 2px solid #ffbd00;
height: 50px;
</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cam2_btn">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>400</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 2px solid #ffbd00;
height: 50px;
</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip2_text">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>400</y>
      <width>450</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>192.168.0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>FileName.mp4</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="status_label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>711</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_btn">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>40</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 2px solid #ffbd00;
height: 50px;
</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="frame1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>256</width>
      <height>480</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Frame1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="frame2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>256</width>
      <height>480</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Frame1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
